01-rstudio-intro.Rmd

```diff
@@ -76,6 +75,15 @@ setx HOME "%USERPROFILE%"
 Press [Enter], you should see **SUCCESS: Specified value was saved**.
 Quit command prompt by typing `exit` then pressing [Enter]
 
+**If you do not have administrative rights on your computer:**
+
+Download R and RStudio from the BC Government Software Centre:
+
+- Install RTools (Rtools 4.0 Gen PO) - this will install RStudio
+- Install R (RforWindowsX64 4.0.5 Gen P0)
+
+
+
 ## Introduction to RStudio
 
```

Comment:
There was a request to add something like this to the end of the "Introduction to Studio" section:
Pro Tip: RStudio will automatically save your session variables in a file called .RData.  These are saved when you exit a project and restored when you open it up again.  If you don't plan to use this feature you can toggle it in the Project Options -> General tab.